src/uct/api/uct_def.h

```diff
@@ -405,4 +406,41 @@ typedef ucs_status_t (*uct_tag_unexp_rndv_cb_t)(void *arg, unsigned flags,
                                                 const void *rkey_buf);
 
 
+/**
+ * @ingroup UCT_RESOURCE
+ * @brief Tuning parameters for an UCT endpoint created by @ref uct_ep_create_sockaddr.
+ */
+typedef struct uct_ep_sockaddr_params {
+    /**
+     * Mask of valid fields in this structure, using bits from
+     * @ref uct_ep_sockaddr_params_field. Fields not specified in this mask
+     * would be ignored. Provides ABI compatibility with respect to adding new
+     * fields.
+     */
+    uint64_t                          field_mask;
+    /**
+     * The sockaddr to connect to on the remote peer. Mandatory filed for
+     * a client.
+     */
+    const ucs_sock_addr_t             *sockaddr;
+    /**
+     * Interface to create the endpoint on. Mandatory filed for a client.
```

Comment:
filed->field

Reply:
can these params be used for a server? If no, would be enough to say that `it is mandatory field`.